aten/src/ATen/native/cuda/ForeachFunctors.cuh

```diff
@@ -316,7 +319,7 @@ struct UnaryOpFunctor {
             }
             else {
                 for(int i_start = 0; i_start < n && i_start < chunk_size; i_start += blockDim.x * kILP) {
-                    load_args<depth>(r_args, args, i_start, chunk_size, n);
+                    load_args<r_args_depth>(r_args, args, i_start, chunk_size, n);
```

Comment:
it's always 1, right?

Reply:
Yes, but r_args_depth is used to declare r_args, so i wanted to be symmetric.